gnd/src/main/java/com/google/android/gnd/repository/OfflineAreaRepository.java

```diff
@@ -186,4 +211,27 @@ public Completable addAreaAndEnqueue(LatLngBounds bounds) {
                     .filter(tileSource -> tileSource.getState() == TileSource.State.DOWNLOADED)
                     .collect(toImmutableSet()));
   }
+
+  /** Delete an offline area and the unique tile sources associated with it. */
```

Comment:
A better explanation might be "Delete an offline area, including any tile sources that don't overlap with other existing offline areas."

Reply:
b86a87a